lib/sdk/storage.js

```diff
@@ -109,10 +137,21 @@ async function getSource(paramsSource, inDirectory, disableSourceDownload) {
     return getAsset(paramsSource, inDirectory, name, disableSourceDownload);
 }
 
-async function putRendition(rendition) {
+async function putRendition(rendition, directories) {
     // Note: validation has happened in validate.js before
 
     if (process.env.WORKER_TEST_MODE) {
+        if (rendition.directory === directories.postprocessing) {
```

Comment:
In which cases do we arrive here? 
I thought that we didn't have postprocessing in the pipeline yet (which is totally on purpose, as pipeline should eventually replace it).

Reply:
same for this one, also from https://github.com/adobe/asset-compute-sdk/pull/139. I can remove it if it is not necessary.

Reply:
Pipeline doesn't do post-processing currently (and should not ever), so I'd prefer to have it removed.

Reply:
removed